src/main/java/domain/LottoStore.java

```diff
@@ -17,9 +18,33 @@ public int getLottoCount() {
         return lottoCount;
     }
 
-    public LottoTickets getLottoTickets(NumberGenerator numberGenerator) {
-        return new LottoTickets(IntStream.range(0, lottoCount)
+    public void validateManualCount(int manualCount) {
+        if (manualCount > lottoCount) {
+            throw new IllegalArgumentException("금액으로 구입할 수 있는 로또의 매수를 초과하였습니다.");
+        }
+        if (manualCount < 0) {
+            throw new IllegalArgumentException("양수를 입력해주세요.");
+        }
+    }
+
+    private LottoTickets buyRandomLottoTickets(NumberGenerator numberGenerator, int manulCount) {
+        return new LottoTickets(IntStream.range(0, lottoCount - manulCount)
                 .mapToObj(number -> new LottoTicket(numberGenerator.generateNumbers()))
                 .collect(Collectors.toList()));
     }
+
+    private LottoTickets buyManualLottoTickets(List<String> userInput) {
+        LottoTickets manualLottos = new LottoTickets();
+
+        userInput.stream().forEach(manualLotto ->
+                manualLottos.addTicket(new LottoTicket(manualLotto)));
+
+        return manualLottos;
+    }
+
+    public LottoTickets buyLottoTickets(NumberGenerator numberGenerator, List<String> userInput) {
```

Comment:
```suggestion
    public LottoTickets buyLottoTickets(NumberGenerator numberGenerator, List<String> manualLottoNumbers) {
```
파라미터 네이밍을 조금더 의미있게 작성하면 가독성 좋은 코드가 될수 있어요 😄